optuna/visualization/parallel_coordinate.py

```diff
@@ -86,13 +86,21 @@ def _get_parallel_coordinate_plot(study, params=None):
         all_params = set(params)
     sorted_params = sorted(list(all_params))
 
+    if any(len(p_name) > 10 for p_name in sorted_params):
+        labelangle = 30
+        labelside = "bottom"
+    else:
+        labelangle = 0
+        labelside = "top"
```

Comment:
How about always setting the angle to 30 and side to "bottom", no matter the params. The reasons is that the labels will overlap with this logic if there are too many hyperparameters that are all shorter than 11 characters.

Reply:
If you are busy with https://github.com/optuna/optuna/pull/1037, maybe we could merge this PR as is and I could create a follow up. If this change sounds reasonable that is.

Reply:
@hvy I'm fine with always setting the angle to 30 and side to "bottom", thanks for suggestion.